examples/pypi-source-deps/pixi.toml

```
@@ -0,0 +1,29 @@
+[project]
+name = "pypi-source-deps"
+version = "0.1.0"
+description = "Add a short description here"
+authors = ["Tim de Jager <tim@prefix.dev>"]
+channels = ["conda-forge"]
+platforms = ["osx-arm64", "linux-64", "win-64"]
+
+[tasks]
+hello = "python -c 'from minimal_project import module;module.hello()'"
+
+[dependencies]
+python = "*"
+
+[pypi-dependencies]
+
+rich = "~=13.7"
+# Lets start with some git dependencies
+# With ssh
+flask = { git = "ssh://git@github.com/pallets/flask" }
+# With https
+requests = { git = "https://github.com/psf/requests.git", rev = "0106aced5faa299e6ede89d1230bd6784f2c3660" }
+# ...or use branch = '' to specify a branch or commit
+
+# You can also directly add a source dependency from file
+minimal_project = { path = "./minimal_project" }
```

```suggestion
minimal_project = { path = "./minimal-project" }
```

How about calling the python package `minimal-project` instead? Most packages that I've seen use `-` instead of `_` in the package name 😅